Cargo.toml

```
@@ -2,6 +2,15 @@
 name = "CreateProcessW"
 version = "0.1.0"
 edition = "2021"
+license = "MIT or Apache-2.0"
+description = "Create and handle processes on Windows using the Win32 API"
+repository = "https://github.com/yozhgoor/CreateProcessW"
+homepage = "https://github.com/yozhgoor/CreateProcessW"
+documentation = "https://docs.rs/CreateProcessW"
+readme = "README.md"
+keywords = ["windows", "processes", "command", "library"]
+categories = ["api-bindings", "development-tool::ffi", "os::windows-apis"]
```

development-tool::ffi definitely not
api-bindings :ok_hand: exactly that
os windows api good I think :+1: 


```suggestion
categories = ["api-bindings", "os::windows-apis"]
```